beacon_node/eth2-libp2p/src/config.rs

```diff
@@ -1,89 +1,130 @@
 use clap::ArgMatches;
+use enr::Enr;
 use libp2p::gossipsub::{GossipsubConfig, GossipsubConfigBuilder};
 use serde_derive::{Deserialize, Serialize};
-use types::multiaddr::{Error as MultiaddrError, Multiaddr};
+use std::path::PathBuf;
+use std::time::Duration;
+
+/// The beacon node topic string to subscribe to.
+pub const BEACON_PUBSUB_TOPIC: &str = "beacon_block";
+pub const BEACON_ATTESTATION_TOPIC: &str = "beacon_attestation";
+//TODO: Implement shard subnets
```

Comment:
Nitpick: not sure if we need to leave "TODO"s for phase 1 stuff

Reply:
Will remove